### PR TITLE
fix choice param parse

### LINKED ISSUE #1522: 使用parameters 动态生成pipeline 参数运行后，pipeline的编辑界面报错
现象描述：
使用了parameters动态生成pipeline参数；
第一次通过 编辑界面将pipeline 代码写入；然后执行后
再次打开 流水线页面报错

pipeline代码：
```
pipeline {
  agent {
    node {
      label 'nodejs'
    }
  }
  
  parameters{
    choice(name: 'DeployType', choices: ['完全发布', '制作镜像', '应用上线', '配置发布'], description:'请选择发布类型')
  }
  
  stages {
  
    stage("test"){
      steps{
            echo "test"  
      }

    }
    
  }
}
```

image

### PR DESCRIPTION
**Pull request by runzexia**
Signed-off-by: runzexia <runzexia@yunify.com>

**What type of PR is this?**

/kind bug
/area devops
/milestone 2.1.1
/assign
/cc @soulseen @zryfish 
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1522

### CODE REVIEW

File: pkg/models/devops/project_pipeline.go [hunk 1]
```diff
@@ -375,16 +376,47 @@ func (s *Parameters) fromEtree(properties *etree.Element) *Parameters {
 				*s = append(*s, &Parameter{
 					Name:         param.SelectElement("name").Text(),
 					Description:  param.SelectElement("description").Text(),
-					DefaultValue: param.SelectElement("name").Text(),
+					DefaultValue: param.SelectElement("defaultValue").Text(),
 					Type:         ParameterTypeMap["hudson.model.PasswordParameterDefinition"],
 				})
 			case "hudson.model.ChoiceParameterDefinition":
+				/* case1
```
[the remark this answers is not in the document]
Comment (runzexia) [reply pkg/models/devops/project_pipeline.go]: fix

**Comment by zryfish** [comment]
ci failed, code not formatted

**Comment by runzexia** [comment]
> ci failed, code not formatted

fix

**Pull request merged**